runtime/src/main/java/org/corfudb/runtime/view/stream/IStreamView.java

```diff
@@ -37,12 +37,6 @@
      */
     void reset();
 
-    /**
-     * Garbage collect all the trimmed entries on this stream
```

Comment:
I'm still not sure I understand how we GC all the compacted addresses of this stream over time, once this is part of the old state of a stream. Can you point me out where we take care of this?

Reply:
This is taken care of in the most recent patch where addresses before compaction mark is trimmed in stream view.